wrappers/clang-target-wrapper.sh

```diff
@@ -8,6 +8,7 @@ if [ "$TARGET" = "$BASENAME" ]; then
     TARGET=$DEFAULT_TARGET
 fi
 ARCH="${TARGET%%-*}"
+TARGET_OS="${TARGET*-%%}"
```

Comment:
This doesn't look right. I think `TARGET_OS="$(TARGET##*-}"` should do the right thing, but please try it out.